citrination_client/client.py

```diff
@@ -478,6 +479,81 @@ def get_pif(self, dataset_id, uid, version = None):
         else:
             return self._get_content_from_url(self.api_url + '/datasets/' + str(dataset_id) + '/version/' + str(version) + '/pif/' + str(uid))
 
+    def submit_experimental_design(self, data_view_id, num_candidates, target, effort, constraints=[], sampler="default"):
+
+        body = {
+            "num_candidates": num_candidates,
+            "target": target,
+            "effort": effort,
```

Comment:
The effort should be less than 30.  Could you assert that and raise an error to the user if its not?

Reply:
👍